### ISSUE TITLE
zoom feature

### ISSUE THREAD

**Issue by whimsicaldreamer** [osvaldasvalutis/imageLightbox.js#2]
Although it was intended to be minimalistic but still is it possible to add a tap to zoom feature in it.? If so how?

**Comment by chiotis** [comment]
I agree and would love to see a pinch zoom in mobile devices. Any way we can help?

**Comment by jayusa123** [comment]
I agree. Pinch zoom capability seems pretty necessary for a "touch-friendly image lightbox". Looks great otherwise, though.

**Comment by LebCit** [comment]
I also agree. Pinch zoom would be a great addition to this minimalistic already awesome plugin !
kudos to @osvaldasvalutis

**Comment by LeoCheung777** [comment]
Agree, with the pitch zoom, this plugin is perfect!

**Comment by MaryGr** [comment]
Oh yes, a zoom function on mobile would make it perfect! How to accomplish that then...
